README.md

```diff
@@ -60,7 +56,7 @@ created Zarr store.*
 1. *`HARMONY_HOST_URL` is the harmony base url for your target environment. e.g. `SIT` would be `https://harmony.sit.earthdata.nasa.gov`*
 
 
-### test in a Browswer:
+### Test in a Browser:
```

Comment:
Ha - I totally saw this show up tangentially in your last PR (in one of the lines near a change you made) and meant to fix it. Nice catch.